src/components/TeaserFeed/docs.md

```diff
@@ -123,16 +138,12 @@ Add an excerpt of the content using the highlight prop (if `highlightLabel` is o
 ```react
 <TeaserFeed
   title='The quick brown fox jumps over the lazy dog'
-  highlight='Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores.'
-  highlightLabel='My highlight'
+  description='Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores.'
```

Comment:
Above this example it says:
> Add an excerpt of the content using the highlight prop (if `highlightLabel` is omitted no label will be displayed).

And here you remove the `highlight` and `highlightLabel`. Add another example below?

I now see why you chose `focus`—I'd still suggest to go with `highlighted` (whole thing). `highlight` & `highlightLabel` something in it.